android/src/main/java/com/getcapacitor/plugin/privacyscreen/PrivacyScreen.java

```diff
@@ -12,10 +16,24 @@
      * Called when the plugin is first constructed in the bridge.
      * This method sets the FLAG_SECURE flag to treat the content of the window as secure,
      * preventing it from appearing in screenshots or from being viewed on non-secure displays.
-     * @see <a href="https://developer.android.com/reference/android/view/WindowManager.LayoutParams#FLAG_SECURE">Android Developer</a>
+     * @see <a href="https://developer.android.com/reference/android/view/WindowManager.LayoutParams#FLAG_SECURE">Android Developers</a>
      */
     public void load() {
-        AppCompatActivity activity = getActivity();
-        activity.getWindow().addFlags(WindowManager.LayoutParams.FLAG_SECURE);
+        Window window = getActivity().getWindow();
+        window.addFlags(WindowManager.LayoutParams.FLAG_SECURE);
```

Comment:
It would probably be reasonable to extract this logic into a private method like `initFlags` or `setupFlags`. In case of any flags that might be added in the future you only have to adjust one code block and could be sure that `load` and `enable` add the same flags / works the same.

Reply:
Good catch!